src/mapping.conf

```diff
@@ -180,7 +180,7 @@
         "ups.realpower.nominal" :       "max_power",
         "ups.power.nominal"     :       "power.default.nominal",
         // same for ePDU
-        "input.power.nominal"   :       "power.default.nominal",
+        "input.realpower.nominal":      "power.default.nominal",
```

Comment:
as per my comment by mail...
please leave the existing mapping, but possibly commented out.
and add the new with "real" on both sides, i.e.
"input.realpower.nominal":      "realpower.default.nominal",

To be applied to 2.1.0 too, thx

Reply:
done